requirements.txt

```diff
@@ -65,3 +65,6 @@ uritemplate==4.1.1
 urllib3==1.26.15
 uvicorn==0.21.1
 whitenoise==6.4.0
+celery==5.3.4
```

Comment:
It looks like the latest versions for these packages are:
celery: 5.5.3
redis: 6.2.0
django-celery-beat: 2.8.1

Is there a reason we're using earlier versions? Are there breaking changes or conflicting dependencies that we need to worry about?